aliases/git/nu_alias_git.nu

```diff
@@ -0,0 +1,202 @@
+#
+# Aliases
+# (sorted alphabetically)
+#
+
+export alias g                     = git
```

Comment:
There is an admittedly obscure nushell built-in `g` to manage separate user shells. Might not be worth shadowing if you typically interact through the specific aliases for the common operations and use the fully qualified names when doing the more obscure `git` operations (or copying them from stackoverflow :P)

Reply:
oof... ._.

Reply:
I removed this alias.